website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -163,23 +109,84 @@ Note that `toggleText` is still required, because it supplies the `aria-label` f
+`\{{on "click" this.myAction}}`). Internally, the component uses the [Hds::Interactive](/utilities/interactive/) utility component.
 
-#### ToggleIcon with other icons
+#### Rendering a button
+
+If you add an event handler (no `@href` or `@route`), a `<button>` element will be rendered:
+
+```handlebars
+{{!-- The Doc::ListContainer component is just to help the component render properly --}}
+<Doc::ListContainer class="hds-dropdown-list">
+  <Hds::Dropdown::ListItem::Interactive {{on "click" this.myAction}} @text="Run command" />
+</Doc::ListContainer>
+```
+
+#### Rendering a link with `@href`
 
-In this example, we have a settings icon with a title, description, separator, and links.
+!!! Info
+
+**Internal Link?**
 
-Note that `toggleText` is still required, because it supplies the `aria-label` for the toggle button.
+When using the `@href` argument, you’re indicating an external link (instead of a route). So, a few relevant HTML attributes are added–`target="_blank"` and `rel="noopener noreferrer"`. However, if the `@href` really _does_ point to an internal link or uses a different protocol (e.g., `mailto` or `ftp`), pass `@isHrefExternal={{false}}` to the component and it will not add any extra HTML attributes.
+!!!
+
+If you pass a `@href` argument, a link (`<a>` element) will be generated:
+
+```handlebars
+{{!-- The Doc::ListContainer component is just to help the component render properly --}}
+<Doc::ListContainer class="hds-dropdown-list">
+  <Hds::Dropdown::ListItem::Interactive @href="https://www.hashicorp.com/request-demo/terraform" @text="Request a demo" />
+</Doc::ListContainer>
+```
+
+#### Rendering a LinkTo (with `@route`)
+
+Pass a `@route` to render Ember `<LinkTo>`. If the route is external to your current engine, pass `@isRouteExternal={{true}}` to use `<LinkToExternal>` instead of `<LinkTo>`. For more details see the [Hds::Interactive component](/utilities/interactive/) documentation. All the standard arguments for the `<LinkTo/LinkToExternal>` components are supported (e.g., `models/model/query/current-when/replace`).
+
+```handlebars
+{{!-- The Doc::ListContainer component is just to help the component render properly --}}
+<Doc::ListContainer class="hds-dropdown-list">
+  <Hds::Dropdown::ListItem::Interactive @route="my.page.route" @model="my.page.model" @text="Activate cluster" />
+</Doc::ListContainer>
+```
+
+#### With a loading "interactive" item
+
+There may be use cases when it’s necessary to put an item in a "loading" state while the app performs some operations (e.g., asynchronously checking the user’s permissions to execute a certain operation once the Toggle’s been clicked).
+
+Pass the argument `@isLoading={{true}}` to the item. This will show a "loading" icon (even if an argument `@icon` is provided) and sets the item as non-interactive until the value of `@isLoading` is set to `false` again.
```

Comment:
```suggestion
Pass the argument `@isLoading={{true}}` to the item. This will show a “loading” icon (even if an argument `@icon` is provided) and sets the item as non-interactive until the value of `@isLoading` is set to `false` again.
```